src/backend/joanie/core/utils/__init__.py

```diff
@@ -218,3 +220,8 @@ class Echo:
     def write(self, value):
         """Write the value by returning it, instead of storing in a buffer."""
         return value
+
+
+def generate_random_code():
+    """Generate a random code."""
+    return get_random_string(18)
```

Comment:
What is the probability to get twice the same code ?

Reply:
Pretty low, I believe. Do you think we should use uuid strings instead ?

Reply:
Nope. We could assume the probability of conflict is pretty low and if one day we encount this error, we should think about a dedupe logic IMO.